js/bookmark.js

```diff
@@ -0,0 +1,22 @@
+export function configureToggleableBookmark() {
+  const toggleBookmark = document.querySelector('[data-js="bookmark-tag"]');
+  console.log(toggleBookmark);
+  console.log(toggleBookmark.innerText);
+
+  const showBookmarkTagged = document.querySelector(
+    '[data-js="card__bookmark--active"]'
```

Comment:
i think to remember that we don't need to use the BEM-method for data-js as designation